README.md

```diff
@@ -62,11 +62,35 @@ There are totally 19 workloads in HiBench. The workloads are divided into 6 cate
 
 3. Logistic Regression (lr)
 
-    This workload benchmarks Logistic Regression implemented in Spark-MLLib examples. Logistic Regreesion is realized with LBFGS. The input data set is generated by LabeledPointDataGenerator based on random balance decision tree. It contains three different kinds of data types, including categorical data, continuous data, and binary data.
+    This workload benchmarks Logistic Regression (lr) implemented in Spark-MLLib. Logistic Regreesion is realized with LBFGS. The input data set is generated by LabeledPointDataGenerator based on random balance decision tree. It contains three different kinds of data types, including categorical data, continuous data, and binary data.
 
 4. Alternating Least Squares (als)
 
-    This workload benchmarks Alternating Least Squares implememnted in Spark-MLLib examples. The input data set is generated by RatingDataGenerator for a product recommendation system.
+    This workload benchmarks Alternating Least Squares implememnted in Spark-MLLib. The input data set is generated by RatingDataGenerator for a product recommendation system.
+
+5. Gradient Boosting Tree (gbt)
+
+    This workload benchmarks Gradient Boosting Tree implememnted in Spark-MLLib. The input data set is generated by GradientBoostingTreeDataGenerator.
+
+6. Linear Regression (lir)
+
+    This workload benchmarks Linear Regression (lir) implemented in Spark-MLLib. Linear Regreesion is realized with SGD. The input data set is generated by LinearRegressionDataGenerator.
```

Comment:
This workload benchmarks Logistic Regression with SGD optimizer (based on RDD-API).